recipes/snorkel-metal/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "snorkel-metal" %}
+{% set version = "0.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a0317439bd7269d9275472a5f4b09b91d907ea4b3be7bc0858e1ec01c55421c4" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  skip: True  # [py<36]
+
+requirements:
+  build:
+    - python=3.6
+    - setuptools
+  run:
+    - python=3.6
+    - matplotlib
+    - networkx=2
+    - nose
```

Comment:
Is `nose` required to run the package or is it just for testing?

Reply:
Nope; just necessary for testing. It's been removed.